backend/src/test/scala/com/softwaremill/bootzooka/test/TestDependencies.scala

```diff
@@ -26,4 +29,11 @@ trait TestDependencies extends BeforeAndAfterAll with TestEmbeddedPostgres { sel
         ._1
     }
   }
+
+  private lazy val backendStub: SttpBackend[IO, Any] =
```

Comment:
again naming: we already have a stub in line 23. Maybe we could call this a `serverStub` for example? Just to be clear what the role of this value is